styles.css

```diff
@@ -0,0 +1,389 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+html,
+body {
+  font-size: 18px;
+  font-family: "Poppins", sans-serif;
+  width: 100%;
+  overflow-x: hidden;
+}
+
+.navbar {
+  background-color: white;
+}
+
+.navbar-content {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-left: 40px;
+  margin-right: 40px;
+  margin-top: 40px;
+  border: none;
+}
+
+.navbar-content img {
+  width: 20px;
+  height: 13px;
+}
+
+.logo {
+  color: #6070ff;
+  font-weight: 700;
+  text-decoration: none;
+}
+
+.nav_link ul {
+  display: none;
+}
+
+.section-1 {
+  height: 720px;
+  background-color: #f7f7f9;
+  border-style: none;
+}
+
+.section-1-container {
+  background-image: url("images/header-shapes\ mobile.png");
+  background-repeat: no-repeat;
+  background-position: center;
+  border-bottom-left-radius: 100px;
+  background-color: white;
+  padding: 160px 25px;
+  width: 100%;
+  height: 100%;
+}
+
+.heading {
+  font-size: 2.5em;
+  font-weight: 700;
+  line-height: 52px;
+}
+
+.leading {
+  font-size: 18px;
+  text-align: start;
+  margin: 20px 0;
+}
+
+.social-heading {
+  color: #6070ff;
+  margin-bottom: 8px;
+}
+
+.socials {
+  display: flex;
+  list-style: none;
+}
+
+.socials img {
+  padding-right: 15px;
+}
+
+.section-2 {
+  display: grid;
+  grid-template-columns: 1fr;
+  align-items: center;
+  justify-content: center;
+  background-color: #f7f7f9;
+  padding-bottom: 100px;
+}
+
+.container-1 img {
+  padding: 20px 20px;
+  width: 100%;
+}
+
+.container-1 {
+  background-color: white;
+  margin-left: 1.2em;
+  margin-right: 1.2em;
+  justify-self: center;
+  margin-top: 100px;
+  border-radius: 20px;
+  border: 1px solid #dfe1e6;
+  padding-bottom: 20px;
+}
+
+.items ul {
+  display: flex;
+  font-size: 12px;
+  align-items: center;
+  justify-content: flex-start;
+}
+
+.items {
+  padding-left: 20px;
+  margin-right: 50px;
+}
+
+.items h1 {
+  font-size: 2em;
+}
```

Comment:
- [ ] The color for the project cards title should be `#172B4D` according to the figma design
- [ ] [OPTIONAL] For better semantics I would recommend to not use an `h1` tag here but use use heading levels (`h2`, `h3`, etc.) . as generally, we use only a single `h1` tag per section this helps to create a clear and organized structure for the content on the page and makes it easier for search engines to understand the hierarchy and relevance of the content.